client/gutenberg/extensions/tiled-gallery/save.jsx

```diff
@@ -1,82 +1,28 @@
-/** @format */
-
-/**
- * External dependencies
- */
-import { RichText } from '@wordpress/editor';
-import classnames from 'classnames';
-
 /**
  * Internal dependencies
  */
+import Layout from './layout';
 import { defaultColumnsNumber } from './edit';
-import GalleryGrid from './gallery-grid';
-import { getActiveStyleName } from './layouts';
-
-export default ( { attributes } ) => {
-	const {
-		align,
-		className,
-		columns = defaultColumnsNumber( attributes ),
-		imageCrop,
-		images,
-		linkTo,
-	} = attributes;
-
-	const layout = getActiveStyleName( className );
-
-	const renderGalleryImage = index => {
-		if ( ! images[ index ] ) {
-			return null;
-		}
-
-		const image = images[ index ];
-
-		let href;
+import { getActiveStyleName } from 'gutenberg/extensions/utils';
+import { LAYOUT_STYLES } from './constants';
 
-		switch ( linkTo ) {
-			case 'media':
-				href = image.url;
-				break;
-			case 'attachment':
-				href = image.link;
-				break;
-		}
+export default function TiledGallerySave( { attributes } ) {
+	const { images } = attributes;
 
-		const img = (
-			<img
-				alt={ image.alt }
-				className={ classnames( {
-					[ `wp-image-${ image.id }` ]: image.id,
-				} ) }
-				data-id={ image.id }
-				data-link={ image.link }
-				src={ image.url }
-			/>
-		);
+	if ( ! images.length ) {
+		return null;
+	}
 
-		return (
-			<figure>
-				{ href ? <a href={ href }>{ img }</a> : img }
-				{ layout !== 'circle' &&
-					image.caption &&
-					image.caption.length > 0 && (
-						<RichText.Content tagName="figcaption" value={ image.caption } />
-					) }
-			</figure>
-		);
-	};
+	const { className, columns = defaultColumnsNumber( attributes ), imageCrop, linkTo } = attributes;
 
 	return (
-		<GalleryGrid
-			align={ align }
-			className={ `wp-block-jetpack-tiled-gallery ${ className }` }
+		<Layout
+			className={ className }
```

Comment:
To be safe, we probably should set default rectangular layout class `is-style-rectangular` here somehow on first render when this is `undefined`. (https://github.com/Automattic/wp-calypso/pull/29559/files#r242806286)

This won't affect functionality anyhow at editor side because `is-style-rectangular` doesn't have any affect on styling, but on view side we pick the layout from that classname.